### PR TITLE
Implemented desktop version of About Us page

### LINKED ISSUE #1483: [Desktop][Mobile] : Update the About Us page.
As a user,
When I click o the About Us button on the main men bar,
Then I should be able to see all the information about FightPandemics organization along with 2 CTAs: "Get Involved" and "Go to Help Board" buttons.

- When I click on the "Get Involved" button,
Then a popup should appear with 3 buttons: "Join as Volunteer", "Join as Ambassador" and "Join Student Program"
(These there buttons should redirect the user on the following links which should open on a new page: 
**"Join as Volunteer":** https://apply.workable.com/fightpandemics,  
**"Join as Ambassador":** https://apply.workable.com/fightpandemics/j/46D6EF3B44/, 
 **"Join Student Program":**  https://apply.workable.com/fightpandemics/j/58B157AAB2/)

- When I click on the "Go to Help Board" button,
Then I should be redirected to the Help Board.

Design Desktop: https://zpl.io/29W9pjl
Design Desktop with popup: https://zpl.io/25WmL9n
Design Mobile: https://zpl.io/aROKRRN
Design Mobile with popup: https://zpl.io/2GZ5L09

**Note:** 
1. All the content on the page should me same as the designs linked above. 
2. All the email addresses on the page should be clickable and when clicked should prompt to a email application.
3. All the social media logos are clickable and when clicked should redirect to that particular social media page on a new tab. Below are the links:

- LinkedIn: https://www.linkedin.com/company/fightpandemics

- Facebook: https://www.facebook.com/FightPandemics

- Twitter: https://twitter.com/fightpandemics

- Instagram: https://www.instagram.com/fightpandemics

4. All the supporter logos are clickable and when clicked should redirect to that particular page on a new tab. Below are the links:
**Life Time supporters:**
_Algolia: https://www.algolia.com
Hackolade: https://hackolade.com
Lokalise: https://lokalise.com
Miro: http://www.miro.com
SauceLabs: http://saucelabs.com
Sentry: https://sentry.io/welcome
Social Bee: https://socialbee.io
White Source: https://www.whitesourcesoftware.com_
**Current Supporters:**
_Airtable: https://airtable.com
AWS: http://aws.amazon.com
Auth0: http://auth0.com
Figma: https://www.figma.com
Gitkraken: https://www.gitkraken.com
Google Maps: https://maps.google.com
Lambda Test: https://www.lambdatest.com
MongoDb: https://www.mongodb.com
Notion: https://www.notion.so
PagerDuty: https://www.pagerduty.com
Prowly: https://prowly.com/en
Sendgrid: http://sendgrid.com
Seo4Ajax: https://www.seo4ajax.com
Slack: https://slack.com
Supermetrics: http://supermetrics.com
Twilio: http://www.twilio.com
Typeform: http://www.typeform.com
Zeplin: http://zeplin.io
Zendesk: http://www.zendesk.com
Workable: https://www.workable.com_
**Past Supporters:**
_Leypal: https://www.leypal.com
Calendly: https://calendly.com_

### PR DESCRIPTION
**Pull request by sujatha-m**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_
This PR resolves #1483

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix. The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 1]
```diff
@@ -2,12 +2,16 @@ import styled from "styled-components";
 import { mq, theme } from "constants/theme";
 const { royalBlue } = theme.colors;
 
+
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js:6]: remove extra line.

Comment (idjevm) [reply client/src/components/AboutUs/AboutUsContainer.js:6]: I think overall there is a lot of formatting needed here....

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 2]
```diff
@@ -24,28 +28,45 @@ export const AboutUsContainer = styled.div`
...
+  margin-right:493px;
+  margin-top: 15.3px;
+  font-family: Poppins;
+  font-size: 38px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;  
+  letter-spacing: normal;
+  color: var(--color-primary-text);
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Use `styled-componets` and no `classes`, also use `rem` and not `pixels`, use the theme colors, and theme fonts.

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 3]
```diff
@@ -24,28 +28,45 @@ export const AboutUsContainer = styled.div`
...
 
 export const SupportersLogosContainer = styled.div`
-  max-width: 100rem;
-  margin: 0 auto;
+  max-width: 100%;
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(14rem, 0.8fr));
-  grid-gap: 1rem;
+  margin-left: 125px;
+  margin-right: 125px;
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Please use `rem`s not `pixels`

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 4]
```diff
@@ -24,28 +28,45 @@ export const AboutUsContainer = styled.div`
...
+  
   > div {
     display: grid;
     place-items: center;
     &::before {
       // for apsect ratio
       display: block;
       padding-bottom: 100%;
-      grid-area: 1 / 1 / 2 / 2;
+      grid-area: 1 / 1 / 2 / 2 ;
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: apply formatting. You should be able to format these with VScode.

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 5]
```diff
@@ -70,6 +91,134 @@ export const MissionAndVisionBannerContainer = styled.div`
...
+margin: 0 auto;
+padding-top: 45.2px;
+padding-bottom: 13.2px;
+marginLeft: 675.2px;
+marginRight: 3382.2px;
+  width: 12.4x;
+  height: 16.6px;
+  background-color: var(--color-primary-brand-color);
+  color: #425af2;
+}
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Use `theme` colors, fonts, etc, use `rem`s not `px`, use

Comment (idjevm) [reply client/src/components/AboutUs/AboutUsContainer.js]: Same for all of these components. below and above

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 6]
```diff
@@ -86,24 +235,125 @@ export const MobileViewBannerContainer = styled.img`
...
   margin: 0 auto;
 `;
 
-export const SocialContainer = styled.div`
-  align-items: center;
-  justify-content: center;
+export const OurCommunity = styled.img`
+  width: 831.7px;
+  height: 425px;
+  display: inline-block;
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Please use `rem` not `px`

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 7]
```diff
@@ -86,24 +235,125 @@ export const MobileViewBannerContainer = styled.img`
...
+    height: 66px;
+    font-family: Poppins;
+    font-size: 44px;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    display: flex;
+    margin: 0 auto;
+    justify-content: center;
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Apply the same as comments above.

File: client/src/components/Button/HelpBoardButton.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import BaseButton from "./BaseButton";
```
Comment (idjevm) [inline client/src/components/Button/HelpBoardButton.js:2]: There is a lot of code repetition here, with `HelpBoardButton`, `LinkButton`,  `popUpButton`, please. create one button only and then extend from that one if you need to add more things to another button. + same comments apply for all Button Components created.

File: client/src/components/AboutUs/AboutUsContainer.js [hunk 9]
```diff
@@ -2,12 +2,16 @@ import styled from "styled-components";
...
+
 export const AboutUsContainer = styled.div`
   width: 100%;
   align-items: center;
   justify-content: center;
   text-align: center;
+  color:#282828;
   margin: 0 auto;
+  
+  background-color:white;
```
Comment (idjevm) [inline client/src/components/AboutUs/AboutUsContainer.js]: Use `theme` colors.

File: client/src/components/Card/GridCard.js [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+              <Meta
+               style={
+                { "font-family": "WorkSans",
+                  "font-size":"20px",
+                  "width":"286px",
+                  "height":"90px",
+                  "text-align": "center",
+                  "marginLeft": "75px",
+                  "marginTop": "-39px",
+                  "marginRight": "886px",
```
Comment (idjevm) [inline client/src/components/Card/GridCard.js]: Lots of same comments as above, use theme colors, don't use pixels, use Rems, don't use inline styles, use styled-components, format this properly (applies to all files).

**Review by idjevm — CHANGES_REQUESTED** [review]
Lots of changes to be made, also, we don't need to have a `Desktop Version` & and a `Mobile Version`, but a version that is both Desktop & Mobile friendly. Please use media queries to be able to create a fully responsive page.

Let me know if you have any questions.

**Review by idjevm — CHANGES_REQUESTED** [review]
are we using the old logos? or

**Comment by sujatha-m** [comment]
> are we using the old logos? or

most of the logos being used are old except few which have been updated in zeplin

**Review by sheeba1 — COMMENTED** [review]
Had a quick look at 'About Us' page. I feel like there is a lot of white space on the page, especially after 
'Get Involved' button

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 1 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js:6]: done

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 2 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: Have replaced classes with styled-components. 
Replaced pixels with rems as much as possible but due to incompatibility between zeplin values and expected page, have retained pixel values. Same applies to theme fonts.

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 3 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: same comment as above

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 4 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: done

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 5 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: same as previous comment

File: client/src/components/Button/HelpBoardButton.js (hunk 8 above)
Comment (sujatha-m) [reply client/src/components/Button/HelpBoardButton.js:2]: removed repeated code as much.

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 9 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: done

File: client/src/components/Card/GridCard.js (hunk 10 above)
Comment (sujatha-m) [reply client/src/components/Card/GridCard.js]: same as previous comment for rem.have removed majority of inline styles

**Comment by galeza** [comment]
@sheeba1 
I agree. There is lots of space between h1 tags

**Review by rabbi08 — CHANGES_REQUESTED** [review]
Agree with @sheeba1 @galeza , there is lots of white space on the page.

Along with than I have below observations:
1. The video under 'Our Story' is NOT working.
2.  The link to 'Help Board' is working as expected but the focus lands in the middle of the 'Help Board' page. The focus should be on the top of the HelpBoard page.

For above observations refer below GIF:
https://share.getcloudapp.com/geuz6YB8

3. The "Supporters" section need more formatting as per design because of the following observations:
    - The format for 'Lifetime supporters', 'Current supporters' is not matching with the format of 'Past 
       supporters'.
     - Great if we can arrange the icons under the headings. For E.g. for 'Past Supporters' it looks like its 
        separate text from the all the icons.

Refer below screen shot
https://share.getcloudapp.com/Apuk7y2j

**Comment by sujatha-m** [comment]
> Agree with @sheeba1 @galeza , there is lots of white space on the page.
> 
> Along with than I have below observations:
> 
> 1. The video under 'Our Story' is NOT working.
> 2. The link to 'Help Board' is working as expected but the focus lands in the middle of the 'Help Board' page. The focus should be on the top of the HelpBoard page.
> 
> For above observations refer below GIF:
> https://share.getcloudapp.com/geuz6YB8
> 
> 1. The "Supporters" section need more formatting as per design because of the following observations:
>    
>    * The format for 'Lifetime supporters', 'Current supporters' is not matching with the format of 'Past
>      supporters'.
>    * Great if we can arrange the icons under the headings. For E.g. for 'Past Supporters' it looks like its
>      separate text from the all the icons.
> 
> Refer below screen shot
> https://share.getcloudapp.com/Apuk7y2j

@sheeba,@galeza,@rabbi08 ,the spacing between headings and sections in the page has been set as per the requirements specified in zeplin.Please refer to the same at https://zpl.io/29W9pjl

The video under our story is currently planned to be a placeholder and hence will not work.
Help board page design/implementation is not handled as part of AboutUs implementation,only link redirection part is handled by AboutUs.

**Review by jessica-lau — CHANGES_REQUESTED** [review]
I also agree that there is too much white space on the page.

1. The "Get Involved" text in the button doesn't look fully centered. (it seems to be more aligned towards the top than the bottom). I'm not sure if this is a totally separate issue though.

image

2. Clicking on the partnerships@fightpandamics.com link leads to a 404 page. "fightpandamics" is mispelled and should be "fightpandemics".

image

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 2 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: removed pixels throughout and replaced with rems.

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 3 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: done

File: client/src/components/AboutUs/AboutUsContainer.js (hunk 5 above)
Comment (sujatha-m) [reply client/src/components/AboutUs/AboutUsContainer.js]: done

File: client/src/components/Card/GridCard.js (hunk 10 above)
Comment (sujatha-m) [reply client/src/components/Card/GridCard.js]: done

**Comment by kterry1** [comment]
> I also agree that there is too much white space on the page.
> 
> 1. The "Get Involved" text in the button doesn't look fully centered. (it seems to be more aligned towards the top than the bottom). I'm not sure if this is a totally separate issue though.
> 
> image
> 
> 1. Clicking on the [partnerships@fightpandamics.com](mailto:partnerships@fightpandamics.com) link leads to a 404 page. "fightpandamics" is mispelled and should be "fightpandemics".
> 
> image

@sujatha-m You can use `align-items: center;` on that button's class or styled component to fix this.

**Comment by wadehrarshpreet** [comment]
New PR created for same, rewrote most of the code need full review.
New PR link:-
https://github.com/FightPandemics/FightPandemics/pull/1577

**Pull request closed without merging**